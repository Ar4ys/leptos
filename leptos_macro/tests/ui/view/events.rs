use leptos::*;

#[component]
fn Component() -> impl IntoView {}

fn main() {
    view! {
        <Component on:click=|| {} />
    };
}
