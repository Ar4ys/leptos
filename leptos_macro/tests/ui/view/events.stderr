error[E0593]: closure is expected to take 1 argument, but it takes 0 arguments
 --> tests/ui/view/events.rs:8:20
  |
8 |         <Component on:click=|| {} />
  |                    ^^^^^^^^ -- takes 0 arguments
  |                    |
  |                    expected closure that takes 1 argument
  |
help: consider changing the closure to take and ignore the expected argument
  |
8 |         <Component on:click=|_| {} />
  |                             ~~~
